src/main/java/io/wisetime/connector/jira/JiraConnector.java

```diff
@@ -182,11 +188,17 @@ public PostResult postTime(final Request request, final TimeGroup timeGroup) {
     };
 
     final Function<Issue, Issue> createWorklog = forIssue -> {
+      timeGroup.getTimeRows().forEach(row -> {
```

Comment:
Can we handle this using a default value in the template instead? E.g.

```
${timeRow.getDescription()!"N/A"}
```

Reply:
SGTM 👍